### PR TITLE
PAYARA-1862 Wrappers for SQL should support JDBC 4.2

### PR DESCRIPTION
**Pull request by jGauravGupta**
* PAYARA-1862 Wrappers for SQL should support JDBC 4.2

* formatting

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/jdbc/jdbc-ra/jdbc40/pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
                 -->
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <compilerArgument>-Xbootclasspath/p:${project.basedir}/jdbc40.jar</compilerArgument>
```
Comment (smillidge) [inline appserver/jdbc/jdbc-ra/jdbc40/pom.xml]: Portions Copyright comment is missing from this file

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**